alternator/stats.cc

```diff
@@ -96,14 +96,20 @@ stats::stats() : api_operations{} {
                     seastar::metrics::description("number of rows read and matched during filtering operations")),
             seastar::metrics::make_counter("rcu_total", rcu_total,
                     seastar::metrics::description("total number of consumed read units, counted as half units")).set_skip_when_empty(),
-            seastar::metrics::make_counter("wcu_total", wcu_total,
-                    seastar::metrics::description("total number of consumed write units, counted as half units")).set_skip_when_empty(),
+            seastar::metrics::make_counter("wcu_total", wcu_total[wcu_types::PUT_ITEM],
+                    seastar::metrics::description("total number of consumed write units, counted as half units"),{op("PutItem")}).set_skip_when_empty(),
+            seastar::metrics::make_counter("wcu_total", wcu_total[wcu_types::DELETE_ITEM],
+                    seastar::metrics::description("total number of consumed write units, counted as half units"),{op("DeleteItem")}).set_skip_when_empty(),
+            seastar::metrics::make_counter("wcu_total", wcu_total[wcu_types::UPDATE_ITEM],
+                    seastar::metrics::description("total number of consumed write units, counted as half units"),{op("UpdateItem")}).set_skip_when_empty(),
+            seastar::metrics::make_counter("wcu_total", wcu_total[wcu_types::INDEX],
+                    seastar::metrics::description("total number of consumed write units, counted as half units"),{op("Index")}).set_skip_when_empty(),
```

Comment:
I think it's ok, I wanted to keep the label name, and this will hold the WCU that was created because of the index that would be on top of the regular op